openshift-ansible.spec

```diff
@@ -85,8 +85,8 @@ touch %{buildroot}%{_datadir}/ansible/%{name}/roles/contiv/.empty_dir
 %doc README*
 %license LICENSE
 %dir %{_datadir}/ansible/%{name}
-%{_datadir}/ansible/%{name}/inventory/dynamic
-%ghost %{_datadir}/ansible/%{name}/playbooks/common/openshift-master/library.rpmmoved
+%{_datadir}/ansible/%{name}/inventory
+%ghost %{_datadir}/ansible/%{name}/playbooks/common/openshift-master
```

Comment:
It should be already owned by the `playbooks` package as the
```spec
%files playbooks
%{_datadir}/ansible/%{name}/playbooks
```

owns entire playbook directory tree.

Reply:
Weird, it was complaining `%{name}/playbooks/common/openshift-master` is now owned by any package

Reply:
Then it must mean the directory is not installed by the `playbooks` rpm. Did you check that?

Reply:
```
$ rpm -qpl http://download.eng.bos.redhat.com/brewroot/packages/openshift-ansible/3.10.0/0.22.0.git.0.b6ec617.el7/noarch/openshift-ansible-playbooks-3.10.0-0.22.0.git.0.b6ec617.el7.noarch.rpm | grep openshift-master | head
/usr/share/ansible/openshift-ansible/playbooks/openshift-master
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/additional_config.yml
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/certificates.yml
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/config.yml
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/enable_bootstrap.yml
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/private
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/private/additional_config.yml
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/private/certificates-backup.yml
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/private/certificates.yml
/usr/share/ansible/openshift-ansible/playbooks/openshift-master/private/config.yml
```

Reply:
If you check there is no `openshift-master` under https://github.com/openshift/openshift-ansible/tree/master/playbooks/common

Reply:
``playbooks/common/openshift-master`` removed by https://github.com/openshift/openshift-ansible/commit/78f11c8f671015d40a630208b548d0790ec3a823

Reply:
Thanks, removed it, PTAL